src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -89,6 +89,8 @@ public function render(PageBridge $content, $partial = false)
         // get controller and invoke action
         $request = new Request($query, $request, [], $cookies);
         $request->attributes->set('_controller', $content->getController());
+        $request->attributes->set('webspace', $content->getWebspaceKey());
+        $request->attributes->set('locale', $content->getLanguageCode());
```

Comment:
But it still doesn't make any sense? AFAIK there was never a webspace and locale set by us on `$request->attributes`, so I can't see why this is needed at all. Would only understand if we would add `_sulu` here with the attributes, as done in the rest of the PR.